geopandas/io/file.py

```diff
@@ -98,15 +98,21 @@ def to_file(df, filename, driver="ESRI Shapefile", schema=None,
         If specified, the schema dictionary is passed to Fiona to
         better control how the file is written. If None, GeoPandas
         will determine the schema based on each column's dtype
+    crs : dict, default None
+       If specified, the crs dictionary is passed to Fiona to
```

Comment:
the indentation of this line is off by one space (I think you need to include one more space)